pkg/collector/corechecks/cluster/ksm/kubernetes_state.md

```diff
@@ -358,6 +358,12 @@
 `kubernetes_state.service.type`
 : Service types. Tags:`kube_namespace` `kube_service` `type`.
 
+`kubernetes_state.ingress.count`
+: Number of ingresses. Tags:`kube_namespace`.
```

Comment:
It's not mentioned but I guess those metrics have the `kube_cluster_name` tag (like the other metrics defined in this file), right?